### PR TITLE
respect F8_API_SERVER_INSECURE_SKIP_TLS_VERIFY when checking whoami

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @surajssd for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

File: .gitignore [hunk 1]
```diff
@@ -77,7 +77,7 @@ assets/js/
 bindata_assetfs.go
 client/
 swagger/
-tool/cli/
+tool/
```
Comment (aslakknutsen) [inline .gitignore]: not everything in /tool is generated repeatedly..

File: cluster/service.go [hunk 2]
```diff
@@ -147,6 +147,7 @@ func (s *clusterService) refreshCache(ctx context.Context) error {
 			return errors.Wrapf(err, "Unable to resolve token for cluster %v", cluster.APIURL)
 		}
 		// verify the token
+		s.clientOptions = append(s.clientOptions, configuration.WithInsecureSkipTLSVerify())
```
Comment (aslakknutsen) [inline cluster/service.go]: Looks like a very odd location to append this. Move it to the creator of this instead. Or some 'default options list' in constructor or similar.

**Comment by fabric8cd** [comment]
@surajssd snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-595-3`

**Comment by fabric8cd** [comment]
@surajssd snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-595-4`

**Comment by fabric8cd** [comment]
@surajssd snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-595-5`

**Pull request closed without merging**